cognite/neat/rules/models/data_types.py

```diff
@@ -175,17 +190,17 @@ class DateTime(DataType):
     sql = "TIMESTAMP"
 
 
-class DateTimeStamp(DataType):
-    name = "dateTimeStamp"
+class Timestamp(DataType):
+    name = "timestamp"
     python = datetime
     dms = dms.Timestamp
     graphql = "Timestamp"
     xsd = "xsd:dateTimeStamp"
```

Comment:
Def remove "xsd:" from all the types, since we know when .xsd that that is the namespace.
Alt we need almost everywhere to remove prefix "xsd:" from string.